index.js

```diff
@@ -124,10 +130,13 @@ module.exports = {
       // Use policy for test environment if both of these conditions are met:
       // 1. the request is for tests and
       // 2. the build include tests
-      let buildIncludeTests = this.app.tests;
+      const env = process.env.EMBER_ENV;
+      let buildIncludeTests = this.app
```

Comment:
I'm wondering if we actually need to determine if build includes tests. Maybe it's enough to analyze if the request is for tests? Is there a case of legitim requests for `/tests` if app does not include tests?

Reply:
yes, you can do a build and then serve tests off the build directly.